### PR TITLE
Adds detection for various browsers and improves version detection for Ubuntu

### PR DESCRIPTION
**Pull request by liviuconcioiu**

### CODE REVIEW

File: Parser/Client/Browser.php [hunk 1]
```diff
@@ -92,9 +93,11 @@ class Browser extends AbstractClientParser
         'A5' => 'Amerigo',
         'AO' => 'Amigo',
         'AN' => 'Android Browser',
+        '3A' => 'AOL',
```
Comment (Simbiat) [inline Parser/Client/Browser.php:96]: According to https://browserwars.fandom.com/wiki/AOL_Explorer and https://en.wikipedia.org/wiki/AOL_Explorer (which clearly indicates Trident engine) and some screenshots I was able to find, it looks like this should be `AOL Explorer` or, at least `AOL Browser`

File: Tests/fixtures/console.yml [hunk 2]
```diff
@@ -564,3 +564,33 @@
...
+  client:
+    type: browser
+    name: Aplix
+    version: "1.03"
+    engine: ""
+    engine_version: ""
+  device:
+    type: console
+    brand: Sanyo
+    model: ""
```
Comment (sanchezzzhak) [inline Tests/fixtures/console.yml:594]: most likely it's a game console `3DO TRY`

File: Parser/Client/Browser.php [hunk 3]
```diff
@@ -92,9 +93,11 @@ class Browser extends AbstractClientParser
         'A5' => 'Amerigo',
         'AO' => 'Amigo',
         'AN' => 'Android Browser',
+        '3A' => 'AOL Browser',
```
Comment (sanchezzzhak) [inline Parser/Client/Browser.php:96]: AOL Explorer

image

File: regexes/device/consoles.yml [hunk 4]
```diff
@@ -35,10 +35,19 @@ OUYA:
...
   model: 'OUYA'
 
+Sanyo:
+  regex: '.*_SANYO_.*'
+  device: 'console'
+  model: ''
+
 Sega:
-  regex: 'Dreamcast'
+  regex: 'Dreamcast|.*_SEGASATURN_.*'
```
Comment (sanchezzzhak) [inline regexes/device/consoles.yml:44]:
```
regex: 'Dreamcast|Aplix_SEGASATURN'
```
greedy modifiers are best avoided if possible (there is only 1 browser under sega saturn, and then it was accessed via a cartridge/disk)

**Review by sanchezzzhak — APPROVED** [review]

**Pull request merged**